src/ucs/sys/string.c

```diff
@@ -124,23 +124,22 @@ void ucs_memunits_to_str(size_t value, char *buf, size_t max)
     }
 }
 
-char* ucs_sockaddr_str(struct sockaddr *sock_addr, char *ip_str)
+char* ucs_sockaddr_str(struct sockaddr *sock_addr, char *str, size_t max_size)
```

Comment:
need to return "const char*", it's more portable way when returning a literal string